### ISSUE TITLE
question about node features

### ISSUE THREAD

**Issue by SharonLee19** [muhanzhang/pytorch_DGCNN#34]
hello,Muhan. Can you help me to solve this qusstion?
My graph fall into three categories. Both of them have the same structures ,only the featuers of nodes in the graph are different. i want to know your code can do this work?
thx

**Comment by muhanzhang** [comment]
Yes. Your case is a special case of the general graph classification problem. The code should work.

**Comment by SharonLee19** [comment]
thx XD
